cocos/rendering/custom/index.jsb.ts

```diff
@@ -39,6 +39,8 @@ let _pipeline: Pipeline | null = null;
 export const INVALID_ID = 0xFFFFFFFF;
 export const enableEffectImport = true;
 
+const LAYOUT_HEADER_SIZE = 8;
```

Comment:
It is 4 for 8? As `INVALID_ID` is 0xFFFFFFFF, seems it is 4 bytes.

Reply:
4 byte 0xFFFFFFFF + 4 byte version. It's totally 8 bytes.

Reply:
Got it.